benchmarks/benchmark_serving_structured_output.py

```diff
@@ -848,7 +851,7 @@ def main(args: argparse.Namespace):
                             'json', 'json-unique', 'grammar', 'regex',
                             'choice', 'xgrammar_bench'
                         ])
-    parser.add_argument("--json_schema_path",
+    parser.add_argument("--json-schema-path",
```

Comment:
one last unrelated change
```suggestion
    parser.add_argument("--json_schema_path",
```